src/main/java/com/cebanova/gitworkflow/GitWorkflow.java

```diff
@@ -0,0 +1,40 @@
+package com.cebanova.gitworkflow;
+
+public class GitWorkflow {
+    public static void main(String[] args) {
+        // Declaring a list of String variables
+        String step1 = "Perform git stash - to store the uncommitted changes";
+        String step2 = "Perform git checkout master - to move to the local master branch";
+        String step3 = "Perform git fetch command - to fetch the origin branches";
+        String step4 = "Perform checkout -b new_branch_name command - to create and instantly move to a new branch," +
+                "that is identical to local master branch.";
+        String step5 = "Add your changes related to your task";
+        String step6 = "Review your changes and check your code before committing them";
+        String step7 = "Execute git commit using the IntelliJ interface, where you can select what changes you want to commit";
+        String step8 = "Once all the changes are committed, execute a git push command by using CTRL+ALT+K hot keys";
+        String step9 = "The committed changes are pushed to an origin branch with the same name, navigate to GitHub branch";
+        String step10 = "Create a Pull Request from origin/new_branch_name into origin/master";
+        String step11 = "Add the Pull Request details - name, description and request review from your colleagues";
+        String step12 = "If you collect at least 2 approvals from your colleagues the merge the branch";
```

Comment:
Please describe what will happen if someone asks for changes

Reply:
I have added step 13 . Please review it and let me know if any additional changes are needed.